CONTRIBUTING.md

```diff
@@ -1,6 +1,12 @@
 Contribute
 ==========
 
+Creating Issues
+---------------
+Run `terminus cli version` and confirm you are  [running the latest version](https://github.com/pantheon-systems/cli/wiki/Installation) before opening a new issue.
+
+
+
```

Comment:
Could you please scale back the exuberant use of whitespace to a single empty newline? Otherwise :+1: